artsy/eigen
# chore: the chorest chore of all: reanimated upgrade (third try, metaverse edition)

This PR resolves []

### Description



### QA Test Case(s)



screens with animations to QA. make sure things work as they should:
- onboarding welcome
- search, input and cancel
- phone number edit in profile
- animatable header. screens with filters, like a gene page
- MyCollectionSearchBar
- MyCollection
- MyCollectionArtworks



### PR Checklist


### To the reviewers 👀


<details><summary>Changelog updates</summary>

### Changelog updates








#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-



</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md
